xrdsst/controllers/client.py

```diff
@@ -0,0 +1,56 @@
+import urllib3
+
+from cement import ex
+
+from xrdsst.api import ClientsApi
+from xrdsst.api_client.api_client import ApiClient
+from xrdsst.controllers.base import BaseController
+from xrdsst.models import ClientAdd, Client, ConnectionType
+from xrdsst.rest.rest import ApiException
+from xrdsst.resources.texts import texts
+
+
+class ClientController(BaseController):
+    class Meta:
+        label = 'client'
+        stacked_on = 'base'
+        stacked_type = 'nested'
+        description = texts['client.controller.description']
+
+    @ex(help="Add client subsystem", arguments=[])
+    def add(self):
+        urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+        self.add_client(self.load_config())
+
+    def add_client(self, configuration):
+        self.init_logging(configuration)
+        for security_server in configuration["security_server"]:
+            BaseController.log_info('Starting configuration process for security server: ' + security_server['name'])
```

Comment:
Exactly same information gets logged as in InitController, please consider changing a bit to be more clear what is configured here.

Reply:
This is the same everywhere, token, client, cert -- do you have actual suggestion to approach this, before the autoconfiguration command is also implemented?

Reply:
I have not noticed it before. Seems to be the same in init, cert, client and token. In all others except init, please add just one or two words what kind of cofiguration process is being started, for token e.g. "Starting token configuration...", otherwise we just have a lot of similar lines in the logs.

Reply:
Redundancy in autoconfiguration is such that I do not believe any of these messages will survive. However, I amended this ATM.